core/jazz_ui/e2e/page-objects/jazzservices.po.ts

```diff
@@ -119,21 +119,33 @@ export class Jazz {
     return element(by.xpath('//li[@class="x caps" and contains(text(),"logs")]'));
   }
   getFilterIcon(){
-	 return element(by.css('section > env-logs-section > div > div > div.drp-wrp > div:nth-child(2) > div > div > div > div > span'));
+	 return element(by.xpath('//div[@class="filter-icon relative"]'));
```

Comment:
you are switching back to xpath?

Reply:
That css path is not a customized one,for the better performance we can use xpath now,code looks good with tht